backend/JiperBackend/Controllers/UserController.cs

```diff
@@ -61,5 +63,50 @@ public IActionResult Login([FromBody] JObject data)
             
             return Ok(user);
         }
+
+        [HttpPost("addorder")]
```

Comment:
You don't need to change the name of endpoint - I think that "newOrder" could have stayed. I just wanted for you to change the controller mothod name. :)

Reply:
soooo change again?

Reply:
I think so. :)